docs/getting_started/Uninstall.md

```diff
@@ -0,0 +1,14 @@
+# Uninstalling
+
+Cleaning up Pelorus is very simple. Just run
+
+```
+helm uninstall pelorus --namespace pelorus
+helm uninstall operators --namespace pelorus
+oc delete namespace pelorus
+```
+
+If Pelorus was deployed with PVCs, you may want to delete them, because helm uninstall will not remove PVCs. To delete them run
+```
+oc delete pvc --namespace pelorus $(oc get pvc --namespace pelorus -o name)
```

Comment:
Is this necessary? since I run `oc delete namespace`, wont it delete everything on it?

Reply:
we don't want to remove oc delete namespace. Think of an example where the user had some other resources in that namespace and pelorus was installed as addition. In such case we don't want the user to remove other things, meaning we don't recommend deleting namespace when uninstalling one app.

This is different if we use freshly created cluster or namespace for development/CI and we are sure nothing important is there.

Reply:
Hmm alright, agreed. I will remove the command to delete namespace